stock-filters/ChirpVillage/Surface.py

```diff
@@ -9,25 +9,57 @@
 
 from Block import Block
 
+from copy import copy
+
 
 class Surface(object):
 
     def __init__(self, level, box):
+        print("surface.__init__")
```

Comment:
Print

Reply:
for now they should stay because it shows progress through the generation. Easier to debug. Can be removed before submission

Reply:
I'm okay with it